### PR TITLE
Include date_created_gmt in order notes endpoint

### PR DESCRIPTION
**Pull request by claudiosanches**
We forget to update the order notes endpoint.
This change applies only to v2.

### CODE REVIEW

File: includes/api/class-wc-rest-order-notes-controller.php [hunk 1]
```diff
@@ -83,6 +83,87 @@ public function get_items( $request ) {
...
+	 * @param WP_REST_Request $request Request object.
+	 * @return WP_REST_Response $response Response data.
+	 */
+	public function prepare_item_for_response( $note, $request ) {
+		$data = array(
+			'id'               => (int) $note->comment_ID,
+			'date_created'     => wc_rest_prepare_date_response( $note->comment_date ),
+			'date_created_gmt' => wc_rest_prepare_date_response( $note->comment_date_gmt ),
+			'note'             => $note->comment_content,
+			'customer_note'    => (bool) get_comment_meta( $note->comment_ID, 'is_customer_note', true ),
```
Comment (mikejolley) [inline includes/api/class-wc-rest-order-notes-controller.php]: wc_string_to_bool?

File: includes/api/class-wc-rest-order-notes-controller.php [hunk 2]
```diff
@@ -83,6 +83,87 @@ public function get_items( $request ) {
...
+					'readonly'    => true,
+				),
+				'date_created_gmt' => array(
+					'description' => __( "The date the order note was created, as GMT.", 'woocommerce' ),
+					'type'        => 'date-time',
+					'context'     => array( 'view', 'edit' ),
+					'readonly'    => true,
+				),
+				'note' => array(
+					'description' => __( 'Order note.', 'woocommerce' ),
```
Comment (mikejolley) [inline includes/api/class-wc-rest-order-notes-controller.php]: Order note content.

File: includes/api/class-wc-rest-order-notes-controller.php [hunk 3]
```diff
@@ -83,6 +83,87 @@ public function get_items( $request ) {
...
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit' ),
+				),
+				'customer_note' => array(
+					'description' => __( 'Shows/define if the note is only for reference or for the customer (the user will be notified).', 'woocommerce' ),
```
Comment (mikejolley) [inline includes/api/class-wc-rest-order-notes-controller.php]: If true, the note will be shown to customers and they will be notified. If false, the note will be for admin reference only.

**Review by mikejolley — CHANGES_REQUESTED** [review]

File: includes/api/class-wc-rest-order-notes-controller.php (hunk 1 above)
Comment (claudiosanches) [inline includes/api/class-wc-rest-order-notes-controller.php]: We save in the data base only `0` and `1`, but sorry, I did not changed it in this Pull REquest, just included `date_created_gmt`...

**Pull request merged**